Tests/SwiftSyntaxParserTest/Inputs/closure.swift

```diff
@@ -1,3 +0,0 @@
-// A closure without a signature. The test will ensure it stays the same after
```

Comment:
I haven't looked through all these, but I assume you moved ones you thought necessary and deleted otherwise?

Reply:
Yes, most of them became initializers calls to create syntax trees.